src/ast/codegen_llvm.cpp

```diff
@@ -1004,6 +1004,24 @@ void CodegenLLVM::visit(Call &call)
     auto arg = call.vargs->at(0);
     auto scoped_del = accept(arg);
   }
+  else if (call.func == "macaddr")
+  {
+    // MAC addresses are presented as char[6]
+    AllocaInst *buf = b_.CreateAllocaBPFInit(call.type, "macaddr");
+    b_.CREATE_MEMSET(buf, b_.getInt8(0), 6, 1);
```

Comment:
allocainit already inits. I'm not sure that memset is really needed here anyway as we proberead 6 bytes into a 6 byte buffer